src/patternfly/components/FormControl/form-control.scss

```
@@ -263,196 +187,177 @@ $pf-v5-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 2
-    --#{$form-control}--m-calendar--BackgroundUrl: var(--#{$form-control}--m-icon-sprite--m-calendar--BackgroundUrl);
-    --#{$form-control}--m-clock--BackgroundUrl: var(--#{$form-control}--m-icon-sprite--m-clock--BackgroundUrl);
-    --#{$form-control}__select--BackgroundSize: var(--#{$form-control}--m-icon-sprite__select--BackgroundSize);
-    --#{$form-control}__select--BackgroundPositionX: var(--#{$form-control}--m-icon-sprite__select--BackgroundPositionX);
-    --#{$form-control}__select--success--BackgroundPosition: var(--#{$form-control}--m-icon-sprite__select--success--BackgroundPosition);
-    --#{$form-control}__select--m-warning--BackgroundPosition: var(--#{$form-control}--m-icon-sprite__select--m-warning--BackgroundPosition);
-    --#{$form-control}__select--invalid--BackgroundPosition: var(--#{$form-control}--m-icon-sprite__select--invalid--BackgroundPosition);
   }
 
-  @at-root select#{&} {
-    --#{$form-control}--PaddingRight: var(--#{$form-control}__select--PaddingRight);
-    --#{$form-control}--PaddingLeft: var(--#{$form-control}__select--PaddingLeft);
-
-    background-image: var(--#{$form-control}__select--BackgroundUrl);
-    background-position: var(--#{$form-control}__select--BackgroundPosition);
-    background-size: var(--#{$form-control}__select--BackgroundSize);
+  > select {
+    --#{$form-control}--PaddingRight: var(--#{$form-control}--select--PaddingRight);
+    --#{$form-control}--PaddingLeft: var(--#{$form-control}--select--PaddingLeft);
 
     // Firefox's select text has additional padding
     // stylelint-disable-next-line
     @-moz-document url-prefix() {
-      --#{$form-control}--PaddingRight: calc(var(--#{$form-control}__select--PaddingRight) - 1px);
-      --#{$form-control}--PaddingLeft: calc(var(--#{$form-control}__select--PaddingLeft) - 4px);
+      --#{$form-control}--PaddingRight: calc(var(--#{$form-control}--select--PaddingRight) - 1px);
+      --#{$form-control}--PaddingLeft: calc(var(--#{$form-control}--select--PaddingLeft) - 4px);
     }
 
-    &[aria-invalid="true"] {
-      --#{$form-control}--PaddingRight: var(--#{$form-control}__select--invalid--PaddingRight);
-      --#{$form-control}--invalid--BackgroundPosition: var(--#{$form-control}__select--invalid--BackgroundPosition); // drop this declaration and use the __select var directly in the background-position below in breaking change release
-
-      background-image: var(--#{$form-control}__select--BackgroundUrl), var(--#{$form-control}--invalid--BackgroundUrl);
-      background-position: var(--#{$form-control}__select--BackgroundPosition), var(--#{$form-control}--invalid--BackgroundPosition);
-      background-size: var(--#{$form-control}__select--BackgroundSize), var(--#{$form-control}--invalid--BackgroundSize);
+    .pf-m-error {
+      --#{$form-control}--PaddingRight: var(--#{$form-control}--select--m-error--PaddingRight);
```

These vars need `--m-[status]` after the base form control class in the name.

```suggestion
      --#{$form-control}--PaddingRight: var(--#{$form-control}--m-error--select--PaddingRight);
```